### DISCUSSION TITLE
Custom map generation from Lidar point cloud

### DISCUSSION THREAD

**Discussion by das007** [carla-simulator/carla#5716] [General]
Is there any way I can convert the lidar point cloud into a custom map?
Lidar point cloud is generated by stitching multiple lidar scans of a specific region.
@nsubiron / @bernatx  , Can you please help with my above query?

**Comment by RaviBeagle** [comment]
As far as I am aware of:

Autoware Vector Map Builder: https://tools.tier4.jp/
With this tool you should able to create an OpenDrive Map which you import it back into CARLA and create your custom map:
https://carla.readthedocs.io/en/latest/tuto_content_authoring_maps/

and 
Mathworks RoadRunner: https://in.mathworks.com/products/roadrunner.html
I am not very sure about RoadRunner though.
